### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by MagicLex**
Tried to keep it as clean and simple as possible.

## Make sure there is no duplicate PR for this issue

* **Please check if the PR meets the following requirements**
- [ ] Adds tests for the submitted changes (for bug fixes & features)
- [ ] Passes the tests
- [ ] HOPSWORKS JIRA issue has been opened for this PR
- [ ] All commits have been squashed down to a single commit

* **Post a link to the associated JIRA issue**

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,108 +1,112 @@
...
-You can develop and run Python, Spark, and Flink applications on Hopsworks - in Jupyter notebooks, as jobs, or even notebooks as jobs. You can build production pipelines with the bundled Airflow, and even run ML training pipelines with GPUs in notebooks on Airflow. You can train models on as many GPUs as are installed in a Hopsworks cluster and easily share them among users.
+# 🚀 Quickstart
+
+## **APP - Serverless (beta)**
+### → **Go to [app.hopsworks.ai](https://app.hopsworks.ai)**
+Hopsworks is available as a serverless app, simply head to [app.hopsworks.ai](https://app.hopsworks.ai) and register with your **Gmail** or **Github** accounts. You will then be able to run a tutorial or access Hopsworks directly and try yourself. This is the prefered way to first experience the platform before diving into more advanced uses and installation requirements. 
+
+## **Azure, AWS & GCP**
+A [Managed Hopsworks](https://managed.hopsworks.ai) account is required as it is our platform for running Hopsworks and the Feature Store in the cloud. It integrates seamlessly with third party platforms such as Databricks, SageMaker and KubeFlow.
+
```
Comment (SirOibaf) [inline README.md:42]: I think this sentence can be improved. I would state immediately that managed.hopsworks.ai integrates with the customer AWS/Azure/GCP environment.

File: README.md [hunk 2]
```diff
@@ -1,108 +1,112 @@
...
+If you wish to run Hopsworks on your Azure, AWS or GCP environnement, follow one of the following guides in our documentation:
```
Comment (SirOibaf) [inline README.md:43]: typo in environment

File: README.md [hunk 3]
```diff
@@ -1,108 +1,112 @@
...
-To build and deploy swagger on your own Hopsworks instance you can follow the instructions found in 
-[this](https://hopsworks.readthedocs.io/en/stable/user_guide/hopsworks/microservices.html?highlight=swagger#swagger-ui) guide.
-
-
-<a name="who"></a>
-## Who’s behind Hopsworks?
-Hopsworks started as an open-source collaborative project at [KTH University](https://www.kth.se/en), [RISE](https://www.ri.se/en), and has more recently been taken on by [Logical Clocks](https://www.logicalclocks.com/). Several funding bodies have helped contribute to its development including: [European Commission (FP7, H2020)](https://ec.europa.eu/), [EIT](https://eit.europa.eu/), [SSF](https://strategiska.se/), [Vinnova](https://www.vinnova.se/) and [Celtic-Next](https://www.celticnext.eu/).
+# 🎓 Documentation and API
+## **Documentation**
+[Hopsworks documentation](https://docs.hopsworks.ai) includes user-guides, Feature Store documentation, and an Administrator Guide. We also include concepts to help user navigates the abstractions and logics of the feature stores and MLOps in general:
```
Comment (SirOibaf) [inline README.md:65]: user guides, feature store documentation and an administration guide.

File: README.md [hunk 4]
```diff
@@ -1,108 +1,112 @@
...
+- **Feature Store:** [https://docs.hopsworks.ai/3.0/concepts/fs/](https://docs.hopsworks.ai/3.0/concepts/fs/)
+- **Projects:** [https://docs.hopsworks.ai/3.0/concepts/projects/governance/](https://docs.hopsworks.ai/3.0/concepts/projects/governance/)
+- **MLOps:** [https://docs.hopsworks.ai/3.0/concepts/mlops/prediction_services/](https://docs.hopsworks.ai/3.0/concepts/mlops/prediction_services/)
+
+## **APIs**
+Hopsworks API documentation is divided in 3 categories; Hopsworks API covers project level APIs, Feature Store API covers feature groups, training datasets and connectors and finally MLOPs API covers Model Registry, Serving and Deployment.
```
Comment (SirOibaf) [inline README.md:71]: covers feature groups, feature views and connectors. also type mlops with consistent capitalization.

File: README.md [hunk 5]
```diff
@@ -1,108 +1,112 @@
...
+
+## **APIs**
+Hopsworks API documentation is divided in 3 categories; Hopsworks API covers project level APIs, Feature Store API covers feature groups, training datasets and connectors and finally MLOPs API covers Model Registry, Serving and Deployment. 
+- **Hopsworks API** - [https://docs.hopsworks.ai/hopsworks-api/3.0.1/generated/api/connection/](https://docs.hopsworks.ai/hopsworks-api/3.0.1/generated/api/connection/)
+- **Feature Store API** - [https://docs.hopsworks.ai/feature-store-api/3.0.0/generated/api/connection_api/](https://docs.hopsworks.ai/feature-store-api/3.0.0/generated/api/connection_api/)
+- **MLOps API** - [https://docs.hopsworks.ai/machine-learning-api/3.0.0/generated/connection_api/](https://docs.hopsworks.ai/machine-learning-api/3.0.0/generated/connection_api/)
+
+## **Tutorials**
+Most of the tutorials require you to have at least an [app.hopsworks.ai](https://app.hopsworks.ai). You can explore the dedicated [https://github.com/logicalclocks/hopsworks-tutorials](https://github.com/logicalclocks/hopsworks-tutorials) repository containing our tutorials or jump directly in one of the existing use cases:
```
Comment (SirOibaf) [inline README.md:77]: at least an account on

**Review by SirOibaf — CHANGES_REQUESTED** [review]

**Review by MagicLex — COMMENTED** [review]
Updates and fixes

File: README.md [hunk 6]
```diff
@@ -1,108 +1,112 @@
...
-You can develop and run Python, Spark, and Flink applications on Hopsworks - in Jupyter notebooks, as jobs, or even notebooks as jobs. You can build production pipelines with the bundled Airflow, and even run ML training pipelines with GPUs in notebooks on Airflow. You can train models on as many GPUs as are installed in a Hopsworks cluster and easily share them among users.
+# 🚀 Quickstart
+
+## **APP - Serverless (beta)**
+### → **Go to [app.hopsworks.ai](https://app.hopsworks.ai)**
+Hopsworks is available as a serverless app, simply head to [app.hopsworks.ai](https://app.hopsworks.ai) and register with your **Gmail** or **Github** accounts. You will then be able to run a tutorial or access Hopsworks directly and try yourself. This is the prefered way to first experience the platform before diving into more advanced uses and installation requirements. 
+
+## **Azure, AWS & GCP**
+[Managed Hopsworks](https://managed.hopsworks.ai) is our platform for running Hopsworks and the Feature Store in the cloud and integrates direction with the customer AWS/Azure/GCP environment. It alslo integrates seamlessly with third party platforms such as Databricks, SageMaker and KubeFlow.
```
[the remark this answers is not in the document]
Comment (SirOibaf) [reply README.md:41]: it alslo

**Review by SirOibaf — APPROVED** [review]

**Pull request merged**